### DISCUSSION TITLE
Declare specific properties in ProjectTo

### DISCUSSION THREAD

**Discussion by omerdavid** [LuckyPennySoftware/AutoMapper#3795] [NA]
Hey, great tool ,i realy find it usefull...
When using ProjectTo<T>() with Ef core ,
is there away to declare only the proprties to select like in any select statement which can render a thin Sql statement,
rather than explicitly ignore each property that should not be selected ?

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/Queryable-Extensions.html#explicit-expansion
But the idea is to use dedicated DTOs for each query so that's not needed.
